packages/features/ee/round-robin/roundRobinReassignment.ts

```diff
@@ -382,12 +383,13 @@ export const roundRobinReassignment = async ({
 
   // Handle changing workflows with organizer
   if (hasOrganizerChanged) {
-    const workflowReminders = await prisma.workflowReminder.findMany({
+    const scheduledWorkflowReminders = await prisma.workflowReminder.findMany({
       where: {
         bookingUid: booking.uid,
         method: WorkflowMethods.EMAIL,
+        scheduled: true,
+        OR: [{ cancelled: false }, { cancelled: null }],
         workflowStep: {
-          action: WorkflowActions.EMAIL_HOST,
```

Comment:
We need to do it for all email actions -->   `method: WorkflowMethods.EMAIL` does that 

even though email will be sent to the same address for the other actions, the email body and replyTo person changes